dockerfile-image-update-cmdtool/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGithubUtilTest.java

```diff
@@ -41,7 +41,7 @@
     public void testGetGithubUtil() throws Exception {
         githubUtil = Mockito.mock(GithubUtil.class);
         dockerfileGithubUtil = new DockerfileGithubUtil(githubUtil);
-        assertEquals(dockerfileGithubUtil.getGithubUtil(), githubUtil);
+        Assert.assertEquals(dockerfileGithubUtil.getGithubUtil(), githubUtil);
```

Comment:
Why add this in? You're statically importing the class.

Reply:
This is a mistake on my part; not sure how I fat fingered this in...will revert.